src/frontend/ts/state/chessStore.ts

```diff
@@ -130,27 +112,8 @@ const useChessStore = create<ChessState>((set, get) => ({
     }
 
     get().setDisplayGame(archivedGame)
-
-    //  XX: this would save the move data with the archivedGame?
-    //      could help with performace, when it comes to returning to a
-    //      previously scry'd for game?
-    //      but that would require an if statement or something to make,
-    //      that seems silly.
-    //
-    //  set(state => ({ archivedGames: state.archivedGames.set(data.gameID, archivedGame) }))
   },
-  //
-  //
-  //  XX: potentially create a receiveArchive func
-  //      this would construct the ArchivedGameInfo, add to the
-  //      archivedGames map here, and then scry for the game moves.
-  //
-  //  OR  recieveGame can switch based off of data.archived boolean
-  //      just like ChessUpdate does.
-  //
   receiveArchive: async (data: GameInfo) => {
```

Comment:
when we receive a `chess-game` we can't tell whether it is an active or archived game, so a created another function just like `receiveGame`. maybe there is a better way to do this, but this is when the new `archived:` field is set to `true`.